test/conformance/helpers/tracing/traces.go

```diff
@@ -62,29 +64,70 @@ func (t SpanTree) toTestSpanTreeHelper() TestSpanTree {
 	for i := range t.Children {
 		children[i] = t.Children[i].toTestSpanTreeHelper()
 	}
-	return TestSpanTree{
+	tst := TestSpanTree{
 		Kind:                     t.Span.Kind,
 		LocalEndpointServiceName: name,
 		Tags:                     t.Span.Tags,
 		Children:                 children,
 	}
+	tst.SortChildren()
+	return tst
 }
 
 // TestSpanTree is the expected version of SpanTree used for assertions in testing.
 type TestSpanTree struct {
-	Root                     bool
-	Kind                     model.Kind
-	LocalEndpointServiceName string
-	Tags                     map[string]string
+	Note                     string            `json:"aaNote,omitempty"`
```

Comment:
Are these `a` and `aa` prefixes added for sorting? If yes can we have some comments for them?

Reply:
Added a comment, does it make sense?